---
title: "Conceptual Boxes"
author: "steppe"
date: "2023-03-01"
output:
  pdf_document: default
---


Here we create three text boxes for the adjacent manuscript. These serve to illustrate three main ideas which we found difficult to incorporate into the written text. 

1) The Applications of Meta-barcoding with Plants

```{r Applications of metabarcoding in plants, echo = F}

applications <- data.frame(
  'Application' = c(
    'Plant-Animal Interactions',
    'Noxious Weed Detection',
    'Detection of Rare Species', 
    'Forensic Science',
    'Pharmaceutical'
  ),
  'Example' = c(

    'Dietary Preferences of herbivores',
    'Presence of species by hydrologic Basin',
    'Presence of Rare Aquatic Species; Others',
    'Identifying the provenance of materials found at crime scene',
    'Identifying adulterants in wholesale products'
  ),
  'Citation' = c(
    'Soininen et al. 2009; reviewed in Banerjee et al. 2022',
    'Coghlan et al. 2021; Xu et al. 2018',
    'Tsukamoto et al. 2021, reviewed in Banerjee et al. 2022',
    'Allwood et al. 2020',
    'Bell et al. 2022'
  )
)

write.csv(applications, '../data/Applications_Table.csv', row.names = F)
read.csv('../data/Applications_Table.csv') |>
  knitr::kable(caption = 'Applications of Plant Metabarcoding',
             col.names = c('Application', 'Example', 'Citations'))

rm(applications)
```


2) Major Issues and bottlenecks currently related to plant Meta-barcoding.  


```{r Major Issues, echo = F}

bottlenecks <- data.frame(
  
  'Issue' = c(
    'Taxonomic Resolution',
    'Reference Library - Phylogenetic',
    'Reference Library - Spatial',
    'Reference Library Generation', 
    'Uncertainty with Matches',
    'Species Surrogates',
    'False Positives'
  ),
  
  'Approach' = c(
    'A353', 
    'Kew PAFTOL, no phylogenetic biases', 
    'Some bias persists towards Europe',
    'Spatial Modelling; Code within', 
    'Temporal Filter System',
    'Temporal Filter System',
    'Spatial & Temporal Modelling, Jaccard Index, high quality reference loci'
  ),
  
  'Citation' = c(
    c('Coissoc et al. 2016, Kress 2017, Johnson et al. 2023'),  #taxonomic resolution
    c('Kress 2017, Bell et al. 2022, Johnson et al. 2023'),  # reference library phylogenetic
    c('Cheng et al. 2018, Darwin Tree of Life 2022, Lewin et al. 2020, Bell et al. 2021'),  # 'Reference Library spatial
    c('Bell et al. 2022'), #  'Reference Library Generation', 
    c('Bell et al. 2022'), #   'Uncertainty with Matches',
    c('?'), #   'Species Surrogates',
    c('Bell et al. 2021') #   'False Positives'
  )
)


write.csv(bottlenecks, '../data/Bottlenecks_Table.csv', row.names = F)

read.csv('../data/Bottlenecks_Table.csv') |>
  knitr::kable(caption = 'Current Issues Facing Plant Metagenomics',
             col.names = c('Issue', 'Our Approach', 'Issue Citation'))

rm(bottlenecks)
```


3) The Summary of Limitations which each method we utilized contains


```{r Summary of Limitations, echo = F}

limitations <- data.frame(
  
  'Component' = c(
    'Stage 1 Species Filter',
    'Species Distribution Modelling',
    'Phenological Modelling',
    'Database Generation',
    'Read Re-assignment',
    'False Positives',
    'Semi-Quantitative Inference'
  ), 
  
  'Limitation' = c(
    'Test Data', # stage 1 species filter
    'Number of Records; Taxonomically Difficult Groups', # SDM generation
    'Post-Initiation of Climate Change Records', # phenological modelling
    'Adequate Phylogenetic/Spatial Representation', # database generation
    'Discrete Frequentist Data', # read re-assignment
    'Which True Species?', # false positives
    'Genome Size, Pollen Grain Size?' # semi-quantitative inference
  ),
  
  'Paths_Forward' = c(
    'Flash Plant Species Surveys on Plot', 
    'Develop and Disseminate Education Materials; Herbaria Collections', 
    'Advocate Herbarium Collections', 
    'Plant and Fungal Tree of Life; 10kP', 
    'Posterior-Probabilities; Floral Abundance, Nectar/Pollen Nutrition', 
    'Jaccard Index, Plot Abundance',
    'Spike Samples with Reference Materials; Several C Sizes'
  )
)

write.csv(limitations, '../data/Limitations_Table.csv', row.names = F)

read.csv('../data/Limitations_Table.csv') |>
  knitr::kable(caption = 'Limitations Faced and Possible Solutions',
             col.names = c('Method Component', 'Limitations', 'Paths Forward')) 

rm(limitations)
```


4) Summary of errors from the framework approach

```{r}

errors <- data.frame(
  
  'Component' = c(
    'Spatial', 'Molecular (Reference Library)',
    'Molecular Reference (Loci)', 'DNA difficult to extract',
    'Reclassification (Genus)', 'Reclassification (Family)'
  ), 
  
  'Example' = c(
    '"Scabrethia Scabra" (false postiive)', '"Asteraceae" (wrong order)',
    '"Trollium" (false positives, same family as target)', '"Agastache" (Hemiparasites)',
    '"Epilobium"', '"Paxistima myrsinites"'
  ), 
  
  'Correction'  = c(
    'Wyethia arizonica', 'Frasera', 
    'Lupinus sp.', 'Pedicularis',
    'Chamaenerion latifolium?', 'Parnassia palustris'
  )
)

write.csv(errors, '../data/Errors_Table.csv', row.names = F)

read.csv('../data/Errors_Table.csv') |>
  knitr::kable(caption = 'Errors faced with components of the framework',
             col.names = c('Component', 'Example', 'Correction')) 

rm(errors)
```


For example many sequences which mapped to the Asteraceae family, but which was flagged by temporal filters and is present in both *B. nevadensis* Cresson and *B. rufocinctus* Cresson pollen is most likely *Frasera* Walter, which failed extractions for the reference library failed (Appendix 11). A similar likely mismatch could be between what was fide molecular evidence as *Agastache pallidiflora* (A. Heller) Rydb. but where feeding was infrequently observed on *Pedicularis* L., likely due to the Lamiales order being represented by only a single molecular reference species. 

Situations where SDM's led to incorrect results at the species level are evident with classification to *Scabrethia Scabra* (Hooker) W.A. Weber, this match almost certainly representing *Wyethia arizonica* A. Gray (@weber1998new), a species known to be visited by queen bee's via our floral observations. An expected inaccuracy of the classification scheme is in genus level placements, e.g. were *Epilobium* L. (Onagraceae Juss.) spp. were classified. However, given the small size of their flowers in the study area, these results more likely indicate that a species of *Chamaenerion* Seg. (a segregate genus) such as *C. angustifolium* (L.) Scop.  or *latifolium* (L.) Sweet is occasionally utilized, as it supported by limited palynology data. An issue with reclassification within the family level in combination with time included reclassifying *Parnassia palustris* L. to *Paxistima myrsinites* (Pursh) Rafinesque. However, based on flower size it is more likely that the visited taxon was *P. palustris*. 

