---
output:
  pdf_document: default
  word_document: default
---

\begin{flushleft}
Dr. Aaron Ellison \newline
Executive Editor \newline
Methods in Ecology and Evolution \newline
\end{flushleft} 

\hfill March 17^th^, 2024  

Dear Dr. Ellison, 

We are writing to propose an original research article entitled  "*Protocol to increase accuracy and fidelity of pollen meta-genomic barcoding using Angiosperms353: a case study using pollen loads from wild bumble bees*".
 
Many meta-barcoding studies that include plant material are limited due to lack of reliable universal barcodes and the extensive potential candidate list (~350,000 plant species) that requires extensive computational and resource to identify best matches.
In this paper we develop an intuitive approach to generate plant reference sequence databases tailored to specific areas for metabarcoding projects, as well as a lens to critically evaluate metagenomic results using phenological data. 
This spatio-temporal approach utilizes, among other sources - museum and citizen science records, species distribution modelling, and the weibull distribution -  to create baseline environmental data in an approach which is globally applicable

Further, we use bait-capture, via the popular Angiosperms 353 probes, to generate multiple sequence reads to increase the accuracy of species identification.
We believe that the use of the Angiosperms353 probes was novel at the inception of the study, and are aware of several manuscripts utilizing them which will be forthcoming in a variety of journals.

Finally, many studies utilizing metabacoding generate spurious results, which are spatially or temporally incongruities with the focal animal species and plants/ There are few simple and intuitive framework for programmatically assessing and rectifying these issues are lacking. 
In this study, we also provide a framework for meaningfully assessing and contextualizing the meta-barcoding results. 

We have no conflicts of interest to disclose.
